scripts/Microsoft.Xaml.Behaviors.Uwp.Native.nuspec

```diff
@@ -8,58 +8,65 @@
-    <file target="lib\uap10.0\Microsoft.Xaml.Interactions.pdb"                  src="..\out\BehaviorsSDKNative\bin\Win32\Release\Microsoft.Xaml.Interactions\Microsoft.Xaml.Interactions.pdb" />
+    <file target="lib\uap10.0\Microsoft.Xaml.Interactivity.winmd"                    src="..\out\BehaviorsSDKNative\bin\Win32\Release\Microsoft.Xaml.Interactivity\Microsoft.Xaml.Interactivity.winmd" />
+    <file target="lib\uap10.0\Microsoft.Xaml.Interactivity.xml"                      src="..\out\BehaviorsSDKNative\bin\Win32\Release\Microsoft.Xaml.Interactivity\Microsoft.Xaml.Interactivity.xml" />
+    <file target="lib\uap10.0\Microsoft.Xaml.Interactivity.pdb"                      src="..\out\BehaviorsSDKNative\bin\Win32\Release\Microsoft.Xaml.Interactivity\Microsoft.Xaml.Interactivity.pdb" />
+    <file target="lib\uap10.0\Microsoft.Xaml.Interactions.winmd"                     src="..\out\BehaviorsSDKNative\bin\Win32\Release\Microsoft.Xaml.Interactions\Microsoft.Xaml.Interactions.winmd" />
+    <file target="lib\uap10.0\Microsoft.Xaml.Interactions.xml"                       src="..\out\BehaviorsSDKNative\bin\Win32\Release\Microsoft.Xaml.Interactions\Microsoft.Xaml.Interactions.xml" />
+    <file target="lib\uap10.0\Microsoft.Xaml.Interactions.pdb"                       src="..\out\BehaviorsSDKNative\bin\Win32\Release\Microsoft.Xaml.Interactions\Microsoft.Xaml.Interactions.pdb" />
 
-    <file target="lib\uap10.0\Design\Microsoft.Xaml.Interactivity.Design.dll"   src="..\out\BehaviorsSDKNative\bin\AnyCPU\Release\Microsoft.Xaml.Interactivity.Design.dll" />
-    <file target="lib\uap10.0\Design\Microsoft.Xaml.Interactivity.Design.pdb"   src="..\out\BehaviorsSDKNative\bin\AnyCPU\Release\Microsoft.Xaml.Interactivity.Design.pdb" />
-    <file target="lib\uap10.0\Design\Microsoft.Xaml.Interactions.Design.dll"    src="..\out\BehaviorsSDKNative\bin\AnyCPU\Release\Microsoft.Xaml.Interactions.Design.dll" />
-    <file target="lib\uap10.0\Design\Microsoft.Xaml.Interactions.Design.pdb"    src="..\out\BehaviorsSDKNative\bin\AnyCPU\Release\Microsoft.Xaml.Interactions.Design.pdb" />
+    <file target="lib\uap10.0\Design\Microsoft.Xaml.Interactivity.Design.dll"        src="..\out\BehaviorsSDKNative\bin\AnyCPU\Release\Microsoft.Xaml.Interactivity.Design.dll" />
+    <file target="lib\uap10.0\Design\Microsoft.Xaml.Interactivity.Design.pdb"        src="..\out\BehaviorsSDKNative\bin\AnyCPU\Release\Microsoft.Xaml.Interactivity.Design.pdb" />
+    <file target="lib\uap10.0\Design\Microsoft.Xaml.Interactions.Design.dll"         src="..\out\BehaviorsSDKNative\bin\AnyCPU\Release\Microsoft.Xaml.Interactions.Design.dll" />
+    <file target="lib\uap10.0\Design\Microsoft.Xaml.Interactions.Design.pdb"         src="..\out\BehaviorsSDKNative\bin\AnyCPU\Release\Microsoft.Xaml.Interactions.Design.pdb" />
 
```

Comment:
How about we get rid of these now?

Reply:
Well, I guess we can keep them around for a while longer for older versions of VS seeing how much effort you have put into supporting both with this change.

---
In reply to: [382664448](https://github.com/microsoft/XamlBehaviors/pull/172#discussion_r382664448) [](ancestors = 382664448)